alternative-root-project.gradle

```diff
@@ -55,12 +57,4 @@ ext {
 
 apply from: 'build.gradle'
 
-configure(subprojects) {
-    afterEvaluate {
-        tasks.verifyGoogleJavaFormat {
-            exclude '**/build/classes/**'
-        }
-    }
-}
-
 apply plugin: MultiProjectReleasePlugin
```

Comment:
Can we try to just apply this plugin to the main build file? it looks like it's the only difference from build.gradle